tests/test_metadata_utils.py

```diff
@@ -1608,6 +1618,216 @@ def test_add_metadata_and_chunk_examples(self):
+                "<|endoftext|>",
+                "<|endoftext|>",
+                "<|endoftext|>",
+                "<|endoftext|>",
+                "<|endoftext|>",
+            ],
+        )
+
+        self.assertEqual(
+            mapped_ds[6]["attention_mask"],
+            [
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+            ],
+        )
+
+        self.assertEqual(
+            mapped_ds[6]["metadata_mask"],
+            [
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+            ],
+        )
+
```

Comment:
A small nit, in a previous commit I've reformatted the tests in this file so that the lists can stay on one line: https://github.com/bigscience-workshop/metadata/blob/e19b777d9e5c4b47d36b861e084d5590f53ffdf9/tests/test_metadata_utils.py#L556-L560

I would be great to do the same for this new test :slightly_smiling_face: